src/kontrol/__main__.py

```diff
@@ -244,6 +244,10 @@ def exec_prove(
     if isinstance(kore_rpc_command, str):
         kore_rpc_command = kore_rpc_command.split()
 
+    for test_name, _ in tests:
+        if not test_name.startswith('test') or test_name.startswith('testFail'):
+            raise RuntimeError(f"Test name should start with 'test' or 'testFail' to be verified: {test_name}")
```

Comment:
Right, thank you, Tamás! I'll mark these comments as resolved since we decided not to implement it in this way, but I appreciate your comments, I haven't realised neither of these things myself.